src/main/java/at/hannibal2/skyhanni/features/itemabilities/abilitycooldown/ActiveAbility.kt

```diff
@@ -0,0 +1,89 @@
+package at.hannibal2.skyhanni.features.itemabilities.abilitycooldown
+
+import at.hannibal2.skyhanni.data.ClickType
+import at.hannibal2.skyhanni.events.item.ItemAbilityCastEvent
+import at.hannibal2.skyhanni.features.dungeon.DungeonAPI
+import at.hannibal2.skyhanni.features.nether.ashfang.AshfangFreezeCooldown
+import at.hannibal2.skyhanni.utils.LorenzColor
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.TimeUtils.format
+import kotlin.math.floor
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+class ActiveAbility(
+    val type: ItemAbilityType,
+    var text: ItemText? = null,
+
+    private var lastActivation: SimpleTimeMark = SimpleTimeMark.farPast(),
+    var specialColor: LorenzColor? = null,
+    private var lastItemClick: SimpleTimeMark = SimpleTimeMark.farPast(),
+) {
+
+    fun activate(color: LorenzColor? = null, customCooldown: Duration = (type.cooldown)) {
```

Comment:
```suggestion
    fun activate(color: LorenzColor? = null, customCooldown: Duration = type.cooldown) {
```
I don't see why you need these brackets

Reply:
resolved